message/messages.go

```diff
@@ -86,6 +88,10 @@ func (m *inboundMessage) BytesSavedCompression() int {
 	return m.bytesSavedCompression
 }
 
+func (m *inboundMessage) String() string {
+	return m.Message().String()
```

Comment:
I think we probably want to include the `nodeID` + `op` + `m.message` in the output here.